corehq/apps/change_feed/tests/test_data_sources.py

```diff
@@ -161,36 +151,6 @@ def location_data():
         yield [loc.location_id for name, loc in locs.items()]
 
 
-@contextmanager
-def stock_data():
```

Comment:
Is there a sql equivalent of this somewhere?

Reply:
Yes, it's called `ledger_data`